seeing-is-believing.el

```diff
@@ -93,6 +122,13 @@ Optional FLAGS are passed to the seeing_is_believing command."
     (end-of-line)
     (insert " # =>")))
 
+(defun seeing-is-believing~flags ()
```

Comment:
Only to show that it's a "private" function, not meant for the user.  It's a convention I've seen elsewhere, but I'm not dogmatic about it.